test/topology/conftest.py

```diff
@@ -182,12 +182,11 @@ def check_pre_raft(cql):
         return False
     # In Scylla, we check Raft mode by inspecting the configuration via CQL.
     experimental_features = list(cql.execute("SELECT value FROM system.config WHERE name = 'experimental_features'"))[0].value
-    return not '"raft"' in experimental_features
-
+    return 'raft' in experimental_features
```

Comment:
The name of this function "pre_raft" suggests (but perhaps it wasn't documented clearly) that the system has **pre-raft** (i.e., **before** raft) bugs - namely bugs in concurrent schema modification **in a single node cluster**. These bugs do not exist in Cassandra (again, on a single-node cluster) or in post-raft Scylla, hence we returned False when raft is enabled or in Cassandra - and True in Scylla when raft is not enabled.

So the function was correct, albeit confusing.